streisand

```diff
@@ -119,9 +119,25 @@ EOF
   echo; echo; ansible-playbook -i inventories/inventory-existing --extra-vars=@global_vars/vars.yml playbooks/existing-server.yml
 }
 
+# be_careful_friend asks you to pay attention because you're about to do
+# something that might be impossible to undo!
+function be_careful_friend() {
```

Comment:
I think you accidentally duplicated `be_careful_friend`. This seems like the same definition as on L61..67 above